src/components/dropdown-disclosure/components/list-item/index.tsx

```diff
@@ -112,7 +112,13 @@ const DropdownDisclosureLinkItem = ({
 }: DropdownDisclosureLinkItemProps) => {
 	return (
 		<DropdownDisclosureListItem>
-			<Link className={s.link} href={href} rel={rel} target={target}>
+			<Link
+				className={s.link}
+				href={href}
+				rel={rel}
+				target={target}
+				prefetch={false}
```

Comment:
Adding prefetch as `false` so that we don't prefetch a bunch of data that doesn't exist for previous versions.